README.md

```diff
@@ -153,14 +163,18 @@ ignore:
 ### How do I use the map??
 You can either view the map via opening the html file, or by serving it with SimpleHTTPServer (runs on localhost:8000)  
```

Comment:
Everything mentioning `SimpleHTTPServer` should be stripped out since the bot runs its own Flask webserver internally using the `web` plugin. Runs on `localhost:8000` by default.

`userdata.js` stuff is still the same.